documenting.rst

```diff
@@ -1529,3 +1529,9 @@ see the make targets above).
 .. _Sphinx: http://sphinx-doc.org/
 .. _virtualenv: https://virtualenv.pypa.io/
 .. _blurb: https://pypi.org/project/blurb/
+
+
+Translations
+============
+
+There are now several official documentation translations. See section :ref:`21.5. Documentation Translations <experts>` and :PEP:`545`.
```

Comment:
This line needs to be split into two lines.

The purpose of this addition is to properly direct people interested in helping with translations, and to give the rest of us a place to refer such people to.   I think the above should be followed by a link to the doc sig instead of leaving it buried in the PEP.  But Julien should confirm that the PEP is up to date or not and otherwise decide what this section should contain.

Reply:
Thanks for the review, @terryjreedy. I made some edits, based on your comments, feel free to let me know if you think there is anything else I should fix.